web/src/api/moreCast2API.ts

```diff
@@ -176,11 +181,15 @@ export const marshalMoreCast2ForecastRecords = (forecasts: MoreCast2ForecastRow[
  * @param forecasts The raw forecast model data.
  * @returns True if the response is a 201, otherwise false.
  */
-export async function submitMoreCastForecastRecords(forecasts: MoreCast2ForecastRow[]): Promise<boolean> {
+export async function submitMoreCastForecastRecords(
+  wf1Token: string,
+  forecasts: MoreCast2ForecastRow[]
+): Promise<boolean> {
   const forecastRecords = marshalMoreCast2ForecastRecords(forecasts)
   const url = `/morecast-v2/forecast`
   try {
     const { status } = await axios.post<MoreCast2ForecastRecord[]>(url, {
+      wf1Token,
```

Comment:
Rather than naming this parameter wf1token, what do you think about 'hiding' its purpose and just call it token so we don't reveal what system/API the token is for?